plugins/cilium-cni/cilium-cni.go

```diff
@@ -28,7 +28,7 @@ import (
 	"github.com/cilium/cilium/common/addressing"
 	"github.com/cilium/cilium/common/plugins"
 	"github.com/cilium/cilium/pkg/client"
-	"github.com/cilium/cilium/pkg/endpoint"
+	endpointid "github.com/cilium/cilium/pkg/endpoint/id"
```

Comment:
Merge this commit with the id package commit in order not to break git bisecting as the previous commit alone will not compile.